Formula/brainfuck.rb

```diff
@@ -0,0 +1,27 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C."
```

Comment:
No final period needed in the description. Does `brew audit --strict <formula>` not notice this?

Reply:
Nope, I wasn't sure about the period as well. I will remove it.